server/src/main/java/org/jetbrains/bsp/bazel/server/sync/ProjectSyncService.java

```diff
@@ -103,12 +103,12 @@ public ScalacOptionsResult buildTargetScalacOptions(ScalacOptionsParams params)
   }
 
   public ScalaTestClassesResult buildTargetScalaTestClasses(ScalaTestClassesParams params) {
-    var project = projectProvider.get();
+    var project = projectProvider.get(params.getOriginId());
```

Comment:
do we need this?

Reply:
for our purpose probably not, but due to the fact that `ScalaTestClassesParams` has originId I passed them on